### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by SpruceMarcy** [babakks/vscode-go-test-suite#4]
Currently, when attempting to use this extension on windows, tests hang indefinitely. I've had a look and it seems to be because the path provided to the child process is incorrect (it attempts to use paths like `/c:/users/mxmbrook/` rather than `c:\\users\\mxmbrook`), and so that process never closes. This can be fixed by replacing `uri.path` with `uri.fsPath`.

With this fix in place, there is also the issue of `logDest` not being supported in windows. That needs to be removed in order for debugging to work, although running tests as normal works regardless of this.

Changes I made to hack together a working version:
testProvider.ts:376
```
        const cmd = this.adapter.getRunCommand(data, test.uri?.fsPath);
        const testDirectory = dirname(test.uri.fsPath);
```
testProvider.ts:447
```
        const testDirectory = dirname(test.uri.fsPath);
        const cmd = this.adapter.getDebugCommand(data, test.uri.fsPath);
```
testProvider.ts:476
```
               // logDest: logFile.fsPath,
```

**Comment by babakks** [comment]
@SpruceMarcy Thanks for reporting this issue, and also for the solution. I just published the new version v0.0.7, with both fixes. Please upgrade your installation and verify if it's working as intended.

**Comment by SpruceMarcy** [comment]
I've just checked, and 0.07 does indeed fix both running and debugging tests on windows. Thank you so much for the update!
